src/Symfony/Component/Yaml/composer.json

```diff
@@ -23,10 +23,10 @@
         "symfony/polyfill-php81": "^1.22"
     },
     "require-dev": {
-        "symfony/console": "^4.4|^5.0|^6.0"
+        "symfony/console": "^5.4"
     },
     "conflict": {
-        "symfony/console": "<4.4"
+        "symfony/console": "<5.4"
```

Comment:
Can you tell me why we need to update composer.json? 

Is it because the Negatable? Is that related to the exclude feature?

Reply:
It is for the negatable `InputOption::VALUE_NEGATABLE` option.

Reply:
Using a negatable option was suggested here https://github.com/symfony/symfony/pull/39641#discussion_r556372547